src/main/java/net/openhft/chronicle/bytes/VanillaBytes.java

```diff
@@ -600,7 +600,7 @@ public int byteCheckSum(int start, int end)
         @Override
         public Bytes<Underlying> appendUtf8(char[] chars, int offset, int length)
                 throws BufferOverflowException, IllegalStateException, BufferUnderflowException, IllegalArgumentException {
-            ensureCapacity(writePosition() + length * 3L);
```

Comment:
Nice fix. `length` is the length in *chars* not bytes, although `offset` is in bytes I believe. Maybe we need to specify this in javadoc?

Reply:
yeah. you are right. it doesn't look clear... Thinking...

Reply:
May need to make the same fix to net.openhft.chronicle.bytes.UncheckedNativeBytes#appendUtf8